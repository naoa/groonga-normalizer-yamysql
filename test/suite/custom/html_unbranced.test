register normalizers/yamysql

table_create remove_phrases TABLE_PAT_KEY ShortText
load --table remove_phrases
[
{"_key": "<remove_html>"}
]
normalize NormalizerYaMySQLRemovePhrase "<p>これは<b<a>テスト<b>です。</p>これは>片側<テストです。" WITH_TYPES|WITH_CHECKS
