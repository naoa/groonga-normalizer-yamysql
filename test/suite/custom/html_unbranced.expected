register normalizers/yamysql
[[0,0.0,0.0],true]
table_create remove_phrases TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
load --table remove_phrases
[
{"_key": "<remove_html>"}
]
[[0,0.0,0.0],1]
normalize NormalizerYaMySQLRemovePhrase "<p>これは<b<a>テスト<b>です。</p>これは>片側<テストです。" WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "これは<Bテストです。これは>片側<テストです。",
    "types": [
      "hiragana",
      "hiragana",
      "hiragana",
      "symbol",
      "alpha",
      "katakana",
      "katakana",
      "katakana",
      "hiragana",
      "hiragana",
      "symbol",
      "hiragana",
      "hiragana",
      "hiragana",
      "symbol",
      "kanji",
      "kanji",
      "symbol",
      "katakana",
      "katakana",
      "katakana",
      "hiragana",
      "hiragana",
      "symbol",
      "null"
    ],
    "checks": [
      6,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      1,
      1,
      6,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      6,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      7,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      1,
      3,
      0,
      0,
      3,
      0,
      0,
      1,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0
    ]
  }
]
